Asset.php

```diff
@@ -28,14 +28,17 @@ class Asset extends AssetBundle
 	 * @inheritdoc
 	 */
 	public $css = [
-		'redactor.css'
+		'redactor.css',
+		'cropper.css',
+		'http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.0.3/css/font-awesome.min.css'
```

Comment:
Hi there, sorry for my late review. Was busy.
Are you sure that this line will work? If I remember right, it will not work. You need to create new `Asset` and use it in `depends` section.